src/library/Uno.Toolkit.Material/Styles/Controls/v2/BottomTabBar.xaml

```diff
@@ -263,9 +260,9 @@
 		<Setter Property="Background" Value="{ThemeResource MaterialBottomTabBarItemBackground}" />
 		<Setter Property="Foreground" Value="{ThemeResource MaterialBottomTabBarItemForeground}" />
 		<Setter Property="BorderBrush" Value="{ThemeResource MaterialBottomTabBarItemBorderBrush}" />
-		<Setter Property="FontFamily" Value="{StaticResource MaterialMediumFontFamily}" />
-		<Setter Property="FontSize" Value="{StaticResource MaterialBottomTabBarFontSize}" />
-		<Setter Property="FontWeight" Value="{StaticResource MaterialBottomTabBarFontWeight}" />
+		<Setter Property="FontFamily" Value="{ThemeResource MaterialMediumFontFamily}" />
+		<Setter Property="FontSize" Value="{StaticResource LabelMediumFontSize}" />
+		<Setter Property="FontWeight" Value="{StaticResource LabelMediumFontWeight}" />
```

Comment:
not themed?

Reply:
btw, can you confirm whether we should use ThemeResource for every resource defined in material/toolkit from now on? (outside of the obvious within file-level constants, like numerical, templates, styles)